collections/nemo_asr/nemo_asr/parts/manifest.py

```diff
@@ -9,15 +9,16 @@
 
```

Comment:
Is there a reason we are keeping the old manifest class?

Reply:
No.

Reply:
Can you add a deprecation error for the old ManifestBase and ManifestEN classes?

Reply:
What do you mean, exactly? There are no such classes anymore.

Reply:
Redefine the class except the init would just raise a Error saying it has been deprecated, and how to update to the new version.

Reply:
When should we delete this code?
Can I just link people to this PR? I have an example in top commit message.